### ISSUE TITLE
Question about the definitions of several parameters (K_water_cell, K_water_int, K_rbc, K_int_pls, permeability and SA_pls_int)

### ISSUE THREAD

**Issue by jessechao830330** [Open-Systems-Pharmacology/Forum#646]
Dear team,

I am trying to understand all the background equations embedded in the PK-Sim model. I have a few questions about the definition of some of the parameters and equations.
 
1. I really don’t understand what is the definition of the Partition coefficient (interstitial/water) K_water_int and Partition coefficient (intracellular/water) K_water_cell in the Ref[1] for the distribution of the drug instead of the absorption. In all five partition coefficient calculate methods provide in the software [2], all of them discussed either tissue/plasma or blood cell/plasma partition coefficient. However, non of them has shown a partition coefficient of each sub-compartment against water. Do I interpret anything wrong? What’s is the formula to calculate the K_water_cell and K_water_int.
2. I also want to clarify about the Partition coefficient (blood cells/plasma) K_rbc. Is its calculation depends on one of the five partition coefficient calculation methods we selected?  However, how about the Partition coefficient (interstitial/plasma) K_int_pls? Same question as to the previous question that all of the five methods discussed either tissue/plasma or blood cell/plasma partition coefficient but not the interstitial/plasma. Is this K_int_pls simply defined as the fraction of the extracellular drug over the plasma drug?
3. About the permeability (not the Specific intestinal permeability) is defined as the following equation: IsSmallMolecule ? (MWEff *1E9 / 336) ^ (-6) * 10^LogMA / 5 * 1E-4 *1E-1: 0. How do those numbers come from? Is there any referenced article to this equation? 
4. About the surface area (plasma/interstitial) SA_pls_int, It is defined as (V_organ* 1000 / A)^0.75 * B * 1E2 . A and B are different inputs depending on the different organs. What are A and B stand for? How do those numbers come from? Is there any referenced article to this equation?

References:
1. https://static-content.springer.com/esm/art%3A10.1007%2Fs10928-017-9559-4/MediaObjects/10928_2017_9559_MOESM1_ESM.txt
2. https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties

**Comment by StephanSchaller** [comment]
Hi @jessechao830330 ,

excellent questions. ou can find some of the answers in the documentation: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#distribution

1. K_water_cell and K_water_int represent the ratio bound to non-water components in the cellular or interstitial space. Their reciprocal value basically represents the unbound fractions in cell and interstitial space (which is why you then often see expressions like [C]/K_water_cell in equations/formulas that define inputs to cellular processes)

2. Yes, K_int_pls is (simply) defined as the fraction of the extracellular (as interstitial) drug over the plasma drug based on the interstitial/plasma protein and water fractions:

3. See section "How are permeability surface-area (PxSA) products predicted in PK-Sim®?" found below the link above. There are also a couple of references listed.

4. Same as for 3.

Hope this helps,
Stephan

**Comment by jessechao830330** [comment]
Thank you for your explanation. If I have any further question, I will let you know. Take care.

**Comment by DonkeySaddle** [comment]
@StephanSchaller, I'm still unclear as to what exactly the "Partition coefficient (water/container)" parameters represent. When examining the RHS of compartments I have noticed that this parameter multiplied by drug concentration gives drug amount (at least, I think that is correct). Why is that exactly? 

Forgive me if I should have created a new topic, felt like this was related enough where it belonged on the same thread. 

Thanks,

DS

**Comment by StephanSchaller** [comment]
@DonkeySaddle ,

> I have noticed that this parameter multiplied by drug concentration gives drug amount (at least, I think that is correct).
No, this should not convert concentrations to amounts. Do you have an example of such a RHS?

Partition coefficients (PC) reflect the ratio "dissolved" within two different "volume fractions" and are dimensionless (so do not convert dimensions). In this case, the PC describes the theoretical concentration ratio of compound dissolved within water and the "fluid" of a specific container (could be "intracellular" with the cellular space fluid consisting of e.g. water, lipids, and protein).
The theory is that only compound dissolved in water (i.e., unbound) is "available" for enzymatic degradation, hence the division of concentrations within that container with the "Partition coefficient (water/container)".

Hope this helps,
Stephan

**Comment by DonkeySaddle** [comment]
@StephanSchaller,
That makes perfect sense. I don't think I realized part of the RHS of dM/dt can be interpreted as being "unitless". Here's the Matlab code for the RHS of an Alprazolam metabolite in periportal liver intracellular compartment.

`y(818) = 0; % Xanax_Itraconazole_fed|Organism|Liver|Periportal|Intracellular|Alprazolam-CYP3A4-alpha-OH pathway Metabolite

dy(818) =  (P_13610*P_5771*P_10747*P_15916*(P_12486/(P_16403+(P_15916*P_12486))));

# The above equation 
# dM/dt = (CYP3A4_con*Kcat*vol*Part_cof*Alpraz_con)/(Part_cof*Alpraz_con + Km)


P_13610 = CYP3A4|Concentration
P_5771 = Alprazolam-CYP3A4-alpha-OH pathway|kcat_app
P_10747 = Liver|Periportal|Intracellular|Volume
P_15916 = Alprazolam|Partition coefficient (water/container)
P_12486 = Alprazolam|Concentration
P_16403 = Alprazolam-CYP3A4-alpha-OH pathway|Km_app`   

I was thrown by the Alprazolam concentration variables (P_12486) not being multiplied by the liver volume(P_10747). Of course, this could be done, you would just have to also multiply the Km parameter (P_16403) by the liver volume as well. 

Is the interpretation correct?

Thanks,

DS

**Comment by StephanSchaller** [comment]
Hi @DonkeySaddle ,

This is a Michaelis-Menten equation:

`CYP3A4_con * Kcat * vol * Part_cof ` equals to the dimension `amount/time (µmol/min)`
and the rest `Alpraz_con/(Part_cof * Alpraz_con + Km)` is dimensionless as:

P_13610 = CYP3A4|Concentration --> µmol/l
P_5771 = Alprazolam-CYP3A4-alpha-OH pathway|kcat_app --> 1/min
P_10747 = Liver|Periportal|Intracellular|Volume --> liter
P_15916 = Alprazolam|Partition coefficient (water/container) --> unitless
P_12486 = Alprazolam|Concentration --> µmol/l
P_16403 = Alprazolam-CYP3A4-alpha-OH pathway|Km_app --> µmol/l

**Comment by DonkeySaddle** [comment]
@StephanSchaller, Thank you so much!

**Comment by jessechao830330** [comment]
@StephanSchaller Just a quick confirm with you that for the partition coefficient of intracellular : plasma. I selected Rodgers and Rowland method to calculate the partition coefficient. I saw that there are terms of f_ew and Ka_PR in the equation popping up from the PK-Sim software. In my knowledge, these two terms refer to the "Extracellular" part in the tissue. However, this equation is used to calculate the "Intracellular" : plasma partition. Did the software actually extract those two terms of extracellular drug when calculating? Or the equation includes those two term for some reasons? Please correct me if I make any mistake. Thank you. 
I am looking forward to hearing from you.

![image](https://user-images.githubusercontent.com/66793768/110135710-1e995680-7dcf-11eb-879d-f552b9751427.jpeg)
